src/main/resources/css/settingsv2.css

```diff
@@ -0,0 +1,66 @@
+.settingsBorder{
+	
+	-fx-border-color: lightgrey;
+	-fx-border-style: hidden hidden solid hidden;
+	-fx-spacing: 5px;
+	-fx-padding: 0 0 5 0;
+}
+
+.saveCancelHboxBorder{
```

Comment:
can't we use the space here
![image](https://user-images.githubusercontent.com/5772511/202411739-22680d1e-5cb0-4b4d-9b3c-c7001e689302.png)

Reply:
Looks better now  👍